### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sriinampudi**

### CODE REVIEW

File: src/easy/Sri_pangram_day1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def checkIfPangram(self, sentence: str) -> bool:
+        c = 0
+        for i in range (0,26):
+            if(ascii_lowercase[i] not in sentence):
```
Comment (smaranjitghose) [inline src/easy/Sri_pangram_day1.py:5]: I guess you need to import ascii_lowercase from the string library.

Ref: https://docs.python.org/3/library/string.html

**Comment by smaranjitghose** [comment]
# My approach to determining your time complexity:

- For each lookup in the string.ascii_lowercase: O(k)
- The loop runs for a fixed range: near O(1)

so it would be O(n).. i.e. linear in nature.

File: src/easy/Sri_pangram_day1.py (hunk 1 above)
Comment (sriinampudi) [reply src/easy/Sri_pangram_day1.py:5]: yes we have to use 'import string' 
but it worked without importing also

Comment (smaranjitghose) [reply src/easy/Sri_pangram_day1.py:5]: Lol.. that sounds cool 🤣

**Pull request merged**